sequence_python/views/ir_sequence.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="sequence_view" model="ir.ui.view">
+        <field name="model">ir.sequence</field>
+        <field name="inherit_id" ref="base.sequence_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//notebook" position="inside">
+                <page name="code" string="Python Code">
+                    <group>
+                        <field name="use_python_code" />
+                        <field
+                            name="python_code"
+                            widget="ace"
+                            options="{'mode': 'python'}"
+                            placeholder="number"
+                            attrs="{'invisible': [('use_python_code', '!=', False)]}"
```

Comment:
@thomaspaulb functionally this feels the wrong way around; I have to uncheck 'use python' to be able to see my expression and its preview.

Reply:
You were right, I fixed it.